doc/project/checklist.md

```diff
@@ -247,46 +247,55 @@ Ensure all these tests verify back-2back access with zero delays, along with par
 
 ### SIM_ALT_TOOL_SETUP
 
-Verify that the sanity test passes cleanly (with no warnings) with one additional tool apart from the primary tool selected for signoff.
+Verify that the smoke test passes cleanly (with no warnings) with one additional tool apart from the primary tool selected for signoff.
 
-### SIM_SANITY_REGRESSION_SETUP
+### SIM_SMOKE_REGRESSION_SETUP
 
-Sanity regression set up for code health. A small suite of tests identified for running the sanity regression on. If the testbench has more than one compile-time configuration, then a sanity test for each configuration should be ideally selected.
+Smoke regression set up for code health.
+A small suite of tests must be identified for the smoke regression.
+If the testbench has more than one build configurations, then each configuration must have a representative test added to the smoke regression.
 
 ### SIM_NIGHTLY_REGRESSION_SETUP
 
-Nightly regression for running all tests with multiple random seeds (iterations) setup. Selecting the number of iterations for running each test is subjective - it depends on the failure rate and available compute resources. For starters, it is recommended to set iterations to 100 for each test. It may be trimmed down if the compute load is too high. As such, the goal should be to have the nightly regression finish overnight so that the results are available next morning for triage.
+Nightly regression for running all tests with multiple random seeds (iterations) setup.
+Selecting the number of iterations for running each test is subjective - it depends on the failure rate and available compute resources.
+For starters, it is recommended to set iterations to 100 for each test.
+It may be trimmed down if the compute load is too high.
+As such, the goal should be to have the nightly regression finish overnight so that the results are available next morning for triage.
 
 ### FPV_REGRESSION_SETUP
 
 Set up FPV regression by adding the module to `hw/formal/fpv_all` script.
 
 ### SIM_COVERAGE_MODEL_ADDED
 
-- Structural coverage collection model checked in. This specifies what hierarchies and what types of coverage to collect. For example, pre-verified sub-mudules (including some `prim` components pre-verified thoroughly with FPV) can be black-boxed and only IO toggle coverage can be setup for those sub-modules for coverage collection.
-- Functional coverage shell object created - this may not contain coverpoints or covergroups yet, but it is primed for use in post-V2 test development.
-
-### PRE_VERIFIED_SUB_MODULES_V1
-
-Sub-modules that are pre-verified with their own testbenches have already reached V1 or higher stage.
+Structural coverage collection model must be checked in.
+This is a simulator tool specific file that captures what hierarchies and what types of coverage is collected.
```

Comment:
The grammar is a bit wonky here. Either add hyphens or rephrase: "There is a version of this file for each supported simulator... "?

Also, as above, this shouldn't be a separate bullet point.